it/mirror/src/test/java/com/linecorp/centraldogma/it/mirror/git/TestZoneAwareMirrorListener.java

```diff
@@ -63,13 +64,20 @@ public void onDisallowed(Mirror mirror) {}
     @Override
     public void onStart(MirrorTask mirror) {
         logger.debug("onStart: {}", mirror);
-        startCount.merge(key(mirror), 1, Integer::sum);
+        startCount.compute(key(mirror), (k, v) -> {
+            if (v == null) {
+                return new AtomicInteger(1);
+            } else {
+                v.incrementAndGet();
```

Comment:
Question) Doesn't `ConcurrentHashMap` provide an atomic update?

Reply:
That's correct. Changed it without much thought. 😓 Let me revert the changes about `startCount`.